examples/react-app/src/FormElements/Sliders.tsx

```diff
@@ -1,86 +1,124 @@
-        ...prevState,
-        [name]: parseFloat(value),
-      }));
-    }
-    event.preventDefault();
-    event.stopPropagation();
-  };
-
-  const handleSubmit = () => {
-    alert(`Submitted Slider Values
-      Stepped Slider: ${sliders.stepped}
-      Decimal Slider: ${sliders.decimal}
-      Large Value Slider: ${sliders.large}
-    `);
+export type Props = {
+  onUpdate: (newValue: number, label: string) => void;
+  sliders: {
+    [key: string]: number;
   };
+};
 
+export default function SlidersForm(props: Props) {
+  const { sliders } = props;
+  const { onUpdate } = props;
   return (
-    <div>
-      <div style={{ padding: '0px 5px 5px 5px' }}>
-        <h2>Slider Form</h2>
-        <form name={'sliders'} onSubmit={handleSubmit}>
-          <span>Stepped Slider (5)</span>
-          <PlmgSlider
-            name={'stepped'}
-            marks={true}
-            defaultValue={20}
-            thumbLabel={false}
-            step={5}
-            rangeValues={'0, 20, 50, 70, 100'}
-            onValueUpdated={(e) => handleSlider(e)}
-          />
-          <br />
-          <span>Decimal Slider</span>
-          <PlmgSlider
-            name={'decimal'}
-            thumbLabel
-            rangeValues={'0.1, 0.2, 0.3, 0.4, 0.5'}
-            onValueUpdated={(e) => handleSlider(e)}
-          />
-          <br />
-          <span>Large Value Slider</span>
-          <PlmgSlider
-            name={'large'}
-            step={10000}
-            rangeValues={'100000, 200000, 300000, 400000, 500000'}
-            onValueUpdated={(e) => handleSlider(e)}
-          />
-          <br />
-          <input
-            style={{ height: '20px' }}
-            type={'submit'}
-            value={'Submit Sliders'}
-          />
-        </form>
-      </div>
-      <br />
-      <h2>Single Sliders</h2>
-      <span>Marked Slider</span>
-      <PlmgSlider
-        name={'marked'}
-        marks={true}
-        thumbLabel={false}
-        rangeValues={'0, 10, 20, 30'}
-        onValueUpdated={(e) => handleSlider(e)}
-      />
-      <span>{`Current Marked Slider Value: ${sliders.marked}`}</span>
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        padding: '32px',
+        border: '2px solid #ccc',
+        boxShadow: '0px 0px 10px #ccc',
+        marginTop: '32px',
+        marginBottom: '32px',
+      }}
+    >
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          alert(
+            `You have submitted the form with the following values: ${JSON.stringify(
+              sliders
+            )}`
+          );
+        }}
+      >
+        <h2>Sliders</h2>
+        <label>{SLIDERS.decimal.label}</label>
+        <PlmgSlider
+          name={SLIDERS.decimal.label}
+          default={SLIDERS.decimal.default}
+          marks={SLIDERS.decimal.marks}
+          step={SLIDERS.decimal.step}
+          range={SLIDERS.decimal.range}
+          value={sliders.decimal}
```

Comment:
`SLIDERS` & `sliders` are the same identifier it can easily become confused. I would suggest a rename of one of them (or both) to be more explicit to what it is representing.